frontend/src/components/Register.js

```diff
@@ -22,7 +21,7 @@ class Register extends Component {
       name: '',
       password: '',
       password_confirm: '',
-      email: '',
+      username: '',
```

Comment:
The first PR in history to include FE and BE changes :heavy_check_mark: :lock:

Reply:
L E G E N D @melodyfelix